### PR TITLE
enh(Python): add insert end function on singly linked list

### PR DESCRIPTION
**Pull request by banerjeesoumya15**
**PR Checklist:**
- [x] My submission is formatted according to the guidelines in the [contributing guide](https://github.com/MakeContributions/DSA#contribution-guidelines)
- [ ] My addition is on refer on the language README.md file
- [x] My addition does not have a spelling problem
- [ ] My submission has a proper and user-friendly description of the algorithm
- [ ] My submission has the time complexity of the algorithm
- [ ] My submission has sample input-output of the program (NOT FOR PYTHON)

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] New algorithm
- [ ] Optimization in previous algorithms
- [ ] Code style update
- [ ] Refactor
- [ ] Documentation
- [x] Other, please describe:

**Briefly describe the changes in this PR**
Added insertion and deletion functionality in Linked List, in Python.

**Other information:**

### CODE REVIEW

**Comment by banerjeesoumya15** [comment]
Can anyone please approve this PR? Or explain me how I can make this better?

**Comment by banerjeesoumya15** [comment]
Hi,

I've made necessary changes to the code. Let me know if any more change needs to be fixed.

File: algorithms/Python/linked_lists/singly.py [hunk 1]
```diff
@@ -78,6 +78,51 @@ def pop(self):
...
+        temp = self.head
+        while(temp.next is not None):
+            temp = temp.next
+
+        temp.next = node
+        return
+    
+    def deleteData(self, data):
+        '''
+        Delete first occurence of given data
```
Comment (Arsenic-ATG) [inline algorithms/Python/linked_lists/singly.py:101]: typo caught by code-spell : occurence ==> occurrence

```suggestion
        Delete first occurrence of given data
```

**Review by Arsenic-ATG — CHANGES_REQUESTED** [review]

File: algorithms/Python/linked_lists/singly.py [hunk 2]
```diff
@@ -86,5 +131,8 @@ def pop(self):
     ll.push('xyz')
     ll.push(1.1)
     ll.pop()
+    ll.insertEnd('END')
+    print(ll)
+    ll.deleteData(1)
```
Comment (Arsenic-ATG) [inline algorithms/Python/linked_lists/singly.py:136]: What if the element that is to be deleted doesn't exist in the list ?

**Review by Arsenic-ATG — APPROVED** [review]

**Pull request merged**